e2e/scripts/vandoor-prepare.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+
+main() {
+    local sha=
+    sha="$1"
+
+    apt-get update
+    apt-get install curl ca-certificates -y
+
+    curl "https://staging.replicated.app/embedded/embedded-cluster-smoke-test-staging-app/ci/${sha}" -H 'Authorization: 2cHePFvlGDksGOKfTicouqJKGtZ' -o ec-release.tgz
```

Comment:
Should we invalidate this license, create a new one and store it as a Github secret?

Reply:
this license file is stored in the repo
so, maybe?
but I don't think we care that this one is exposed

Reply:
ah i see. ok then.